### DISCUSSION TITLE
Request to for a MAUI project example using Svg.Skia

### DISCUSSION THREAD

**Discussion by DclearEE** [wieslawsoltes/Svg.Skia#113] [Ideas]
Hi, I hope this is the right place to request this.

I was wondering if someone could share a project or show how this can be implemented in a new .NET MAUI project? 
I see that there are a few examples with Avalonia using Svg.Skia, but there is nothing with MAUI using it other than this image: > ![image](https://user-images.githubusercontent.com/2297442/148681465-c30b7585-8867-46f9-88f7-454a68699769.png)
 
Thanks for all your effort with this package!

**Comment by wieslawsoltes** [comment]
This image is from specialized library based on Svg.Model that is shared with Svg.Skia, the Maui library is closed source.

**Comment by wieslawsoltes** [comment]
MAUI does not use SkiaSharp everywhere and it's drawing API is broken, reported many issue and still not fixed.

**Comment by wieslawsoltes** [comment]
Done https://github.com/wieslawsoltes/Svg.Skia/pull/508
